src/lib.rs

```diff
@@ -101,7 +101,7 @@ use std::{
 
 use specta::{
     functions::FunctionDataType,
-    ts::{ExportConfiguration, TsExportError},
+    ts::{ExportConfiguration as SpectaExportConfiguration, TsExportError},
```

Comment:
It would be good to remove this `as SpectaExportConfiguration` and just use `specta::ts::ExportConfiguration` instead of `SpectaExportConfiguration` everywhere else because it's easier to understand on first glance.